internal/clients/clients.go

```diff
@@ -231,10 +231,7 @@ func (c *Cache) Get(cr auth.Credentials, o ...GetOption) (client.Client, error)
 	c.mx.RUnlock()
 
 	if ok {
-		log.Debug("Used existing cached client",
```

Comment:
context of this change is here:

https://github.com/upbound/xgql/pull/130/files#r1383523763